app/client/ts/util.ts

```diff
@@ -19,6 +19,9 @@ export const safeClient: SafeClient =
 
 export const WATCH_URL_RE: RegExp = /\/watch\/([a-zA-Z0-9\/\+]+)/;
 
+// TODO: does not handle international characters
+export const VIDEO_TITLE_RE: RegExp = /^[A-Za-z0-9 ]+$/;
```

Comment:
http://www.regular-expressions.info/unicode.html#prop

It is probably worth testing it out on regex101.com. We could also just have a blacklist of evil chars instead, but I think white-lists are much better for security.